storage/migrations/01_runtimes.up.sql

```diff
@@ -180,12 +184,27 @@ CREATE INDEX ix_runtime_events_nft_transfers ON chain.runtime_events (runtime, (
         type = 'evm.log' AND
         evm_log_signature = '\xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef' AND
         jsonb_array_length(body -> 'topics') = 4;
+
 -- Added in 07_runtime_events_evm_contracts_events.up.sql
 -- Index used for fetching events emitted by a specific contract.
 -- CREATE INDEX ix_runtime_events_evm_contract_events ON chain.runtime_events (runtime, (body ->> 'address'), evm_log_signature, round)
 --     WHERE
 --         type = 'evm.log';
 
+-- Added in 13_runtime_events_related_accounts.up.sql.
+-- CREATE TABLE chain.runtime_events_related_accounts
+-- (
+--     runtime runtime NOT NULL,
+--     round UINT63 NOT NULL,
+--     event_index UINT31 NOT NULL,
+
+--     tx_index UINT31,
+--     type TEXT NOT NULL,
+--     account_address oasis_addr NOT NULL,
```

Comment:
this is a little out of sync with 13_....up.sql, missing e.g. the PRIMARY KEY

Reply:
Updated, thanks for noticing!